src/acmApi/authHelper.ts

```diff
@@ -0,0 +1,31 @@
+import axios, {AxiosInstance, AxiosRequestConfig} from "axios";
+
+function getBaseApi() {
+    return axios.create({
+        baseURL: process.env.REACT_APP_ACMAPI_URL,
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
```

Comment:
I'd like to add in two more headers to designate where the requests are originating from. `User-Agent` and `Host` headers would be useful additions when debugging.

```suggestion
            'Content-Type': 'application/json'
            'User-Agent': 'ACM API Request Handler'
            'Host': 'ACM Portal'
```

Something along these lines ^, doesn't have to be the same thing

Reply:
I think using the `Origin` header might be best here. WDYT?

Reply:
Ah yes, you're right. It should be `Origin`. Mixed up Host & Origin. We can change the Host to be `Portal-Backend` to remove it altogether.